os/arch/arm/src/armv8-m/up_initialstate.c

```diff
@@ -114,12 +114,6 @@ void up_initial_state(struct tcb_s *tcb)
 	/* Set the stack limit value */
 
 	xcp->regs[REG_R10] = (uint32_t)tcb->stack_alloc_ptr + 64;
-#endif
-#ifdef CONFIG_ARMV8M_TRUSTZONE
-	/* By default, a task is created without a secure context */
-	xcp->regs[REG_R8] = 0;
-	/* for stack limit register */
-	xcp->regs[REG_R9] = (uint32_t)tcb->stack_alloc_ptr;
```

Comment:
This is stack limit register setting. Is it okay to remove?

Reply:
ARMv8 has a different stack limit register and we dont have to store this value in R9.